t/01-test-utilities.t

```diff
@@ -51,7 +51,8 @@ subtest 'warnings in sub processes are fatal test failures' => sub {
     my $ipc_run_harness = OpenQA::Test::Utils::_setup_sigchld_handler 'test-process', start sub {
         OpenQA::Test::Utils::_setup_sub_process 'test-process';
         # wait at most 5 seconds (supposed to be interrupted by SIGTERM)
-        sleep 5;
+        sleep(OpenQA::Test::TimeLimit::scale_timeout 5);
```

Comment:
Do we have any other way of catching when the signal should have been received? I guess we can't easily mock it if the code is in a different process?

Thinking out loud anyway... I like the idea of scaling the sleep, though, if that's the best we can do... but I guess it should be 10 at least.

Reply:
5 is already a lot considering it just has to execute a very few Perl code to eventually delete a key from a hash. Of course if the OS does not give the test process any CPU time we're running into the timeout.

Note that the sleep isn't too bad as it gets interrupted as soon as the signal arrives. We don't unconditionally wait 5 seconds here.